app/js/states/inventory/inventory.controller.js

```diff
@@ -209,12 +220,6 @@ function InventoryCtrl(
 
     function parseBrowserQueryParams() {
         FilterService.parseBrowserQueryParams();
-        if (params.sort_field) {
-            InventoryService.setSortField(params.sort_field);
-            if (params.sort_dir) {
-                InventoryService.setSortDirection(params.sort_dir);
-            }
-        }
     }
 
     parseBrowserQueryParams();
```

Comment:
We can probably get rid of the local function parseBrowserQueryParams and replace this line with FilterService.parseBrowserQueryParams();